nexus/lib/pseudopotential.py

```diff
@@ -1956,6 +1971,496 @@ def ppconvert(self,outfile,ref,extra=None):
         execute(command,verbose=True)
         os.system('rm '+tmpfile)
     #end def ppconvert
+
+
+    # test needed
+    def append_to_component(self,l,coeff,expon,rpow):
```

Comment:
Add a 1-2 sentence docstring very briefly describing what this function does.  If the role of this function is simply to be called by another, note that here along with the name of the other function.  Please do the same for the other functions below.